src/commands/download_drive.ts

```diff
@@ -0,0 +1,48 @@
+import { EID } from 'ardrive-core-js';
+import { cliArDriveAnonymousFactory, cliArDriveFactory } from '../index';
+import { CLICommand, ParametersHelper } from '../CLICommand';
+import { CLIAction } from '../CLICommand/action';
+import { DrivePrivacyParameters, DriveIdParameter, LocalPathParameter } from '../parameter_declarations';
+import { getOutputFolderPathAndName } from '../utils';
+import { join as joinPath } from 'path';
+
+new CLICommand({
+	name: 'download-drive',
+	parameters: [DriveIdParameter, { name: LocalPathParameter, description: 'TODO' }, ...DrivePrivacyParameters],
+	action: new CLIAction(async (options) => {
+		const parameters = new ParametersHelper(options);
+		const driveId = parameters.getRequiredParameterValue(DriveIdParameter, EID);
+		const maxDepth = await parameters.getMaxDepth(Number.MAX_SAFE_INTEGER);
+		const destOutputPath = parameters.getParameterValue(LocalPathParameter) || '.';
+		const [destFolderPath, customFolderName] = getOutputFolderPathAndName(destOutputPath);
+		let outputPath: string;
+
+		if (await parameters.getIsPrivate()) {
+			const wallet = await parameters.getRequiredWallet();
+			const ardrive = cliArDriveFactory({
+				wallet,
+				feeMultiple: parameters.getOptionalBoostSetting()
```

Comment:
You can omit this since we're just reading.

Reply:
Nice catch! cd48e8a37b1acb67eaea67c4c0a377258cf5a7cd